src/ConfigWidgets/include/ConfigWidgets/PropertyConfigWidget.h

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef PROPERTY_CONFIG_WIDGET_H_
```

Comment:
We usually prefix this with CONFIG_WIDGETS_

Reply:
Yes. Oversight on my sight.